src/features/code_actions.zig

```diff
@@ -310,7 +352,7 @@ fn createCamelcaseText(allocator: std.mem.Allocator, identifier: []const u8) ![]
 }
 
 // returns a discard string `\n{indent}_ = identifier_name;`
-fn createDiscardText(builder: *Builder, identifier_name: []const u8, declaration_start: usize, add_block_indentation: bool) ![]const u8 {
+fn createDiscardText(builder: *Builder, identifier_name: []const u8, declaration_start: usize, add_block_indentation: bool, add_suffix_newline: bool) ![]const u8 {
```

Comment:
Could you add a comma after the last parameter and then run `zig fmt` so the parameters gets spread out over multiple lines?